doc/dune

```diff
@@ -11,7 +11,7 @@
   (package coq)
   (source_tree sphinx)
   (source_tree tools))
- (action (run sphinx-build -j4 -b html -d sphinx_build/doctrees sphinx sphinx_build/html)))
+ (action (run sphinx-build -j4 -W -b html -d sphinx_build/doctrees sphinx sphinx_build/html)))
```

Comment:
I would prefer we go a bit further and unify this with the Makefile setting by using a variable `%{env:SPHINXOPTS=-j 2 -W}` so that users can disable warnings by overriding it.

Reply:
Sounds good.